plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseTypeMapping.java

```diff
@@ -320,11 +321,11 @@ public void testEnum()
     public void testUuid()
     {
         SqlDataTypeTest.create()
-                // TODO map to Trino UUID
-                .addRoundTrip("UUID", "'417ddc5d-e556-4d27-95dd-a34d84e46a50'", createUnboundedVarcharType(), "CAST('417ddc5d-e556-4d27-95dd-a34d84e46a50' AS varchar)")
-                .execute(getQueryRunner(), clickhouseCreateAndInsert("tpch.test_uuid"));
-
-        // TODO add test with UUID written from Trino
+                .addRoundTrip("UUID", "'114514ea-0601-1981-1142-e9b55b0abd6d'", UuidType.UUID, "cast('114514ea-0601-1981-1142-e9b55b0abd6d' as UUID)")
+                .execute(getQueryRunner(), clickhouseCreateAndInsert("default.ck_test_uuid"));
+        SqlDataTypeTest.create()
+                .addRoundTrip("uuid", "'114514ea-0601-1981-1142-e9b55b0abd6d'", UuidType.UUID, "cast('114514ea-0601-1981-1142-e9b55b0abd6d' as uuid)")
+                .execute(getQueryRunner(), trinoCreateAsSelect("default.ck_test_uuid"));
```

Comment:
We should also test with CREATE + INSERT from Trino. That follows a different code path.

Reply:
Let's also test NULL values.

And it looks to me that the arguments passed to `addRoundTrip` can be reused for both cases (the only difference is lowercase vs uppercase UUID)?